src/base-app.js

```diff
@@ -63,7 +63,7 @@ class FusionApp {
       let {value, aliases} = registered.get(token);
       let provides = value;
 
-      if (value !== null && typeof value === 'object' && value.__plugin__) {
```

Comment:
I believe this line was like this as a result of a convo @AlexMSmithCA @dennisgl and I had about the explicitness of handling primitives

Reply:
Maybe I'm missing something, but I think it is fine to do a truthy check here

Reply:
We opted to be more explicit to better guard against potential edge cases (and I generally prefer to be more explicit in core code).  Change set from yesterday can be found [here](https://github.com/fusionjs/fusion-core/pull/85/files).

Being less explicit allows things we might not want (like a function with `__plugin__` defined) to opt into the plugin-only logic here.  Given that that is fairly unlikely to occur in reality, however, I think either one is fine.

Reply:
I prefer simpler code, and I think this simplifies it a great deal